examples/flutter_gallery/lib/gallery/demos.dart

```diff
@@ -159,6 +159,15 @@ List<GalleryDemo> _buildGalleryDemos() {
       routeName: BackdropDemo.routeName,
       buildRoute: (BuildContext context) => BackdropDemo(),
     ),
+    GalleryDemo(
+      title: 'Banner',
+      subtitle: 'Displaying a banner within a list',
+      icon: GalleryIcons.lists_leave_behind,
+      category: _kMaterialComponents,
+      routeName: BannerDemo.routeName,
+      documentationUrl: 'https://docs.flutter.io/flutter/material/MaterialBanner-class.html',
```

Comment:
Shouldn't our doc links be `https://api.flutter.dev/flutter/...`?

Reply:
looks like all of them use the old url and redirect, I updated this one to the new url.